### DISCUSSION TITLE
Storing unescaped JSON

### DISCUSSION THREAD

**Discussion by calipzo** [spatie/laravel-settings#156] [Q&A]
How can I store unescaped JSON in the database? Tried to implement `toJson` in my GeneralSettings class, but no effect.

```
    public function toJson($options = 0): string
    {
        return json_encode($this->toArray(), JSON_UNESCAPED_UNICODE);
    }
```

Thanks.

**Comment by rubenvanassche** [comment]
You probably should create your own SettingsRepository, it handles the JSON encoding.
